espflash/src/connection/reset.rs

```diff
@@ -199,6 +200,73 @@ impl ResetStrategy for UsbJtagSerialReset {
     }
 }
 
+trait RtcWdtReset {
+    fn wdt_wprotect(&self) -> u32;
+    fn wdt_wkey(&self) -> u32;
+    fn wdt_config0(&self) -> u32;
+    fn wdt_config1(&self) -> u32;
+}
+
+impl RtcWdtReset for crate::targets::esp32c3::Esp32c3 {
```

Comment:
I would really like to avoid leaking chip-specific details like these into other modules, this all probably belongs in the `targets` module.